linuxPackages.bcc build fails

## Issue description

https://hydra.nixos.org/build/89686856/nixlog/1

```
In file included from /build/source/src/cc/frontends/b/parser.h:21:0,
                 from /build/source/src/cc/frontends/b/loader.cc:17:
/build/source/src/cc/frontends/b/lexer.h:22:10: fatal error: FlexLexer.h: No such file or directory
 #include <FlexLexer.h>
          ^~~~~~~~~~~~~
compilation terminated.
```

### Steps to reproduce

Try to install `linuxPackages.bcc` or `linuxPackages_hardened.bcc` on nixpkgs master.

--- a/pkgs/os-specific/linux/bcc/default.nix
+++ b/pkgs/os-specific/linux/bcc/default.nix
@@ -19,7 +19,7 @@ python.pkgs.buildPythonApplication rec {
   buildInputs = [
     llvmPackages.llvm llvmPackages.clang-unwrapped kernel
     elfutils luajit netperf iperf
-    systemtap.stapBuild
+    systemtap.stapBuild flex
   ];
 
   patches = [

